src/domUpdates.js

```diff
@@ -7,6 +7,7 @@ let selectFilteredRooms = document.getElementById('rooms');
 let grabRoomTypeBtn = document.getElementById('grabRoomType');
 let errorHandingLine = document.getElementById('errorHandingLine');
 let userMessage = document.getElementById('userMessage');
+let userMessage2 = document.getElementById('userMessage2');
```

Comment:
Being that your query selectors don't need to change, any that you're assigning out right may be better served as const rather than let. It won't change functionality, but I think it's best practice. Maybe ask about it during your project checkin?